scripts/dev/check_for_enum_scope_usage.py

```diff
@@ -197,10 +197,13 @@ def run(self):
             if len(e.usages) == 0:
                 apparent_enums_in_zero_source_files.append(e.describe())
             unique_files_in_usages: Set[str] = set()
-            for u in e.usages:
-                unique_files_in_usages.add(u.file_path.name)
-            if len(unique_files_in_usages) == 1:
-                apparent_enums_in_only_one_source_file.append(f"{e.describe()} in {next(iter(unique_files_in_usages))}")
+            # exceptions listed by <FILE>:<ENUM NAME>
```

Comment:
If people object to this new convention of beginning enumerations with `e` (I prefer it to ending all enumerations with `Type`) I can get rid of it using a `sed` script after this PR is merged (or give the `sed` script to @dareumnam).

Reply:
I object, for the same reasons I objected to introducing Hungarian notation. IDEs do a really good job now of telling what is what, so in my opinion encoding type information in the name of the variable isn't needed.

Reply:
How do you feel about 40% of the enums in EnergyPlus ending in the word `Type`?

Reply:
I'm not a huge fan, for me it falls into the same category as all the uses of `Data` as a prefix/suffix.

Reply:
I don't have a strong opinion on naming conventions, but why change the pattern now and then add exceptions to the script that's enforcing the existing rule?